web-console/src/lib/components/analytics/TableSqlPrograms.tsx

```diff
@@ -180,7 +180,11 @@ const TableSqlPrograms = () => {
     ApiError,
     { program_id: ProgramId; request: UpdateProgramRequest }
   >({
-    mutationFn: args => ProgramsService.updateProgram(args.program_id, args.request)
+    mutationFn: args => ProgramsService.updateProgram(args.program_id, args.request),
```

Comment:
I'm surprised only TableSqlProgram needs changes, what about the table with connectors?

Reply:
This change in TableSqlPrograms was not necessary, but I think it may help with some corner case in the future. Connector table works as-is, as well